app/controllers/admin/organizations_controller.rb

```diff
@@ -6,10 +6,14 @@ def edit
 
   def update
     @organization = Organization.find(params[:id])
-    if OrganizationUpdateService.update(@organization, organization_params)
+    update = OrganizationUpdateService.update(@organization, organization_params)
+
+    return false unless update
+
+    if update.errors.none?
       redirect_to admin_organizations_path, notice: "Updated organization!"
     else
-      flash[:error] = "Failed to update your organization."
+      flash[:error] = update.errors.full_messages.first
```

Comment:
If there are multiple errors, probably best to include them all? `update.errors.full_messages.join("\n")`